core/templates/pages/feedback-updates-page/feedback-updates-page.component.spec.ts

```diff
@@ -0,0 +1,848 @@
+      expect(component.isCurrentFeedbackSortDescending).toBe(true);
+    });
+
+    it('should sort feedback updates given sorting property as last updated' +
+      ' in ascending order', fakeAsync(() => {
+      // The default sort option for Feedback Updates is last updated.
+      expect(component.currentFeedbackThreadsSortType)
+        .toBe('lastUpdatedMsecs');
+      expect(component.isCurrentFeedbackSortDescending).toBeTrue();
+      expect(component.getValueOfFeedbackThreadSortKey())
+        .toBe('lastUpdatedMsecs');
+
+      fixture.detectChanges();
+
+      const feedbackListNameNodes =
+        fixture.debugElement.nativeElement
+          .querySelectorAll('.e2e-test-feedback-exploration') as
+           NodeListOf<HTMLElement>;
+
+      // The forEach loop is being used here because
+      // getValueOfSubscriptionSortKey is used in a *ngFor directive.
+      // Note that given subscription list is not sorted.
+      feedbackListNameNodes.forEach((titleNode, index: number) => {
+        if (index === 0) {
+          expect(titleNode.innerText).toContain('Biology');
+        }
+        if (index === 1) {
+          expect(titleNode.innerText).toContain('Algebra');
+        }
+        if (index === 2) {
+          expect(titleNode.innerText).toContain('Three Balls');
+        }
+        if (index === 3) {
+          expect(titleNode.innerText).toContain('Zebra');
+        }
+      });
+    }));
+
+    it('should sort feedback updates given sorting property as last updated' +
+      ' in descending order', fakeAsync(() => {
+      // The default sort option for Feedback Updates is last updated.
+      expect(component.currentFeedbackThreadsSortType)
+        .toBe('lastUpdatedMsecs');
+      expect(component.isCurrentFeedbackSortDescending).toBeTrue();
+      expect(component.getValueOfFeedbackThreadSortKey())
+        .toBe('lastUpdatedMsecs');
+
+      component.setFeedbackSortingOptions('lastUpdatedMsecs');
+
+      expect(component.isCurrentFeedbackSortDescending).toBeFalse();
+
+      tick();
+      fixture.detectChanges();
+
+      const feedbackListNameNodes =
+        fixture.debugElement.nativeElement
+          .querySelectorAll('.e2e-test-feedback-exploration') as
+           NodeListOf<HTMLElement>;
+
+      // The forEach loop is being used here because
+      // getValueOfSubscriptionSortKey is used in a *ngFor directive.
+      // Note that given subscription list is not sorted.
+      feedbackListNameNodes.forEach((titleNode, index: number) => {
+        if (index === 0) {
+          expect(titleNode.innerText).toContain('Zebra');
+        }
+        if (index === 1) {
+          expect(titleNode.innerText).toContain('Three Balls');
+        }
+        if (index === 2) {
+          expect(titleNode.innerText).toContain('Algebra');
+        }
+        if (index === 3) {
+          expect(titleNode.innerText).toContain('Biology');
+        }
+      });
+    }));
```

Comment:
Put the `innerText` in an array and then write a expect stmt

Reply:
Already written in codebase please check here(https://github.com/oppia/oppia/blob/develop/core/templates/pages/learner-dashboard-page/learner-dashboard-page.component.spec.ts#L739)

But done according to your suggestion.